DopplerCustomDomain/CustomDomainProvider/TraefikRuleTypeEnum.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DopplerCustomDomain.CustomDomainProvider
+{
+    public enum TraefikRuleTypeEnum
```

Comment:
```suggestion
    public enum TraefikRuleType
```
As it is an enum I don't think it is necessary to specify it, I think

Reply:
I would like to use a name not so tied to the implementation details and more to the behavior, but I cannot think in a better one :(, maybe simply `RuleType`?